gramps/gui/plug/_windows.py

```diff
@@ -1217,7 +1217,7 @@ def install_addons(self, obj):
                      "%s %s" % (ngettext("{number_of} addon was installed.",
                                          "{number_of} addons were installed.",
                                          count).format(number_of=count),
-                                _("You need to restart Gramps to see new views.")),
+                        _("You need to restart Gramps to use the new addons.")),
```

Comment:
> I don't mind you renaming config keys to make their use clearer. Please continue.

Change config keys to mention addon
https://github.com/gramps-project/gramps/pull/306